test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Migration/MigrationServiceTests.cs

```diff
@@ -34,7 +38,34 @@ public async Task TestHandleMigrationAssessmentRequest()
                 MigrationService service = new MigrationService();
                 await service.HandleMigrationAssessmentsRequest(requestParams, requestContext.Object);
                 requestContext.VerifyAll();
-            }     
+            }
+        }
+
+        [Test]
+        public async Task TestHandleMigrationGetSkuRecommendationsRequest()
+        {
+            GetSkuRecommendationsResult result = null;
+
+            var requestParams = new GetSkuRecommendationsParams()
+            {
+                DataFolder = Path.Combine("..", "..", "..", "Migration"),
```

Comment:
nit: maybe place the test data inside a dedicated folder within the 'Migration' folder called 'Data' or something? Just so we don't clutter the main folder with a mix of test classes and sample data